api/RTCPeerConnection.json

```diff
@@ -2054,6 +2106,56 @@
           }
         }
       },
+      "icegatheringstatechange_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/icegatheringstatechange_event",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#event-icegatheringstatechange",
+          "description": "<code>icegatheringstatechange</code> event",
+          "support": {
+            "chrome": {
+              "version_added": "59"
+            },
+            "chrome_android": {
+              "version_added": "59"
+            },
+            "edge": {
+              "version_added": "15"
+            },
+            "firefox": {
+              "version_added": "22"
+            },
+            "firefox_android": {
+              "version_added": "44"
```

Comment:
The Firefox + Firefox for Android data in https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection#browser_compatibility looks pretty implausible, suggesting that the unprefixed `RTCPeerConnection` was available on desktop since 22, but that on Android it was prefixed as `mozRTCPeerConnection` from 22 to 43, and unprefixed in 44.

That's probably wrong, and the likely error is being spread around more here.

https://mdn-bcd-collector.appspot.com/tests/api/mozRTCPeerConnection can be used to figure out when the prefixed support was added and removed, if you have the appetite for it.

Reply:
The unprefixing happened in Firefox 44 in the code: https://bugzilla.mozilla.org/show_bug.cgi?id=1155923. So RTCPeerConnection on Fx 22 is impossible.

Note that the removal of the unprefixed version has not happened yet: https://bugzilla.mozilla.org/show_bug.cgi?id=1531812.

I'm adding a commit reflecting this.